.github/workflows/fossa.yml

```diff
@@ -0,0 +1,21 @@
+name: FOSSA
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '0 0 * * 1'
+
+jobs:
+  fossa-scan:
+    name: License and Security Scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Run FOSSA Scan
+        uses: fossas/fossa-action@v1
+        with:
+          api-key: ${{ secrets.FOSSA_API_KEY }}
```

Comment:
Could you please document how this gets populated and also what the FOSSA scan is for in general?

Reply:
It's a CNCF requirement, so I'll have to do a bit of homework for that. 🙂 Where should this be documented? Like in comments in the workflow? Or maybe in the contributing guide?

Reply:
The contributing guide would be good!

Reply:
Description added--let me know what you think, and I'll update the other contributing guides also.